stories/components/Modal.stories.mdx

```diff
@@ -49,68 +69,261 @@ Nevertheless, the **Modal.Header** will look the same whether you go for option
-    <ModalHeaderBodyAndActionsStory />
-   </Story>
-</Canvas>
-
-
-<Canvas>
-  <Story name="ModalScrollableBodyStory">
-    <ModalScrollableBodyStory />
-   </Story>
-</Canvas>
-
-<Canvas>
-  <Story name="PlainMediumModalStory">
-    <PlainMediumModalStory />
-   </Story>
-</Canvas>
-
-<Canvas>
-    <Story name="ModalPlaygroundStory" args={{
-        plain: false,
-        showCloseIcon: false,
-        medium: false,
-        large: false,
-        title: 'Header of Modal',
-        subtitle: 'This is a smaller description',
-    }}>
-     {Template.bind({})}
-   </Story>
-</Canvas>
\ No newline at end of file
+<Preview withToolbar>
+    <Story
+        parameters={{ docs: { source: { type: 'code' } } }}
+        name="Playground"
+        args={{
+            plain: false,
+            showCloseIcon: false,
+            medium: false,
+            large: false,
+            title: 'Header of Modal',
+            subtitle: 'This is a smaller description',
+        }}
+    >
+        {WithPlaygroundTemplate.bind({})}
+    </Story>
+</Preview>
+
+<Preview withToolbar>
+    <Story
+        parameters={{ docs: { source: { type: 'code' } } }}
+        name="Modal Header"
+        args={{
+            title: 'Header of Modal',
+            buttonTitle: 'Header Only',
+            buttonText: 'Click me to launch a Modal with Header',
+        }}
+    >
+        {ModalTemplate.bind({})}
+    </Story>
+</Preview>
+
+<Preview withToolbar>
+    <Story
+        parameters={{ docs: { source: { type: 'code' } } }}
+        name="Modal Header And Body"
+        args={{
+            body: 'The Body of a Modal can contain whatever you like!',
+            title: 'Header of Modal',
+            subtitle: 'This is a smaller description',
+            buttonTitle: 'Header and Body',
+            buttonText: 'Click me to launch a Modal with Header and Body',
+        }}
+    >
+        {ModalTemplate.bind({})}
+    </Story>
+</Preview>
+
+<Preview withToolbar>
+    <Story
+        parameters={{ docs: { source: { type: 'code' } } }}
+        name="Modal Header Body with Actions"
+    >
+        {WithActionsTemplate.bind({})}
+    </Story>
+</Preview>
+
+<Preview withToolbar>
+    <Story
+        parameters={{ docs: { source: { type: 'code' } } }}
+        name="Modal Scrollable Body"
+        args={{
+            body: ScrollableText,
+            title: 'Header of Modal with Scrollable Body',
+            buttonTitle: 'Scrollable Body',
+            buttonText: 'Click me to launch a Modal with Scrollable Body',
+        }}
+    >
+        {ModalTemplate.bind({})}
+    </Story>
+</Preview>
+
+<Preview withToolbar>
+    <Story
+        parameters={{ docs: { source: { type: 'code' } } }}
+        name="Plain Medium Modal"
+        args={{
+            body: PlainMediumText,
+            title: 'Header of Modal',
+            buttonTitle: 'Header, Body and Actions',
+            buttonText: 'Click me to launch a medium-sized Modal with Header and plain Body',
```

Comment:
Not a knock on the work here, and not something we need to fix right now, but it's a bit weird that the "medium" sized modal at 680px can be larger than a "large"-sized one (60%) (the top one is medium):

![image](https://user-images.githubusercontent.com/8531248/118469625-72fa7400-b738-11eb-93a3-f1645d79af66.png)

Reply:
This is odd - I had a look and the media breakpoint here seems to kick in.

![image](https://user-images.githubusercontent.com/15801768/118517302-6295c980-b737-11eb-9529-3724a3f677a1.png)

When applying `width: calc(~'100% - 80px');` as the width in `&.large` instead of 60%, it works as expected. As this seems to be an issue directly relating to the component as a whole,  I'll raise an issue so that it's tackled separately

Reply:
Issue raised here: https://github.com/Doist/reactist/issues/521